AdblockPlusSafari/AdblockPlusController.m

```diff
@@ -174,3 +187,72 @@ - (void)updateAccessoryViewOfCell:(UITableViewCell *)cell
 }
 
 @end
+
+@implementation AdblockPlusController: AdblockPlusControllerBase
+#ifdef CONFIGURABLE_CUSTOM_FILTER_LIST_ENABLED
```

Comment:
It is quite difficult to add new items to UITableViewController with static cells. I am using this 'hack' (similar which we do use in ABB) to add new items. This solution might not be 100% reliable and it can break with future version of ios, however it is inferencing just only with dev build (removed in production).